man/fi_domain.3.md

```diff
@@ -375,6 +378,15 @@ are defined.
   manual progress may still need application assistance to process
   received operations.
 
+*FI_PROGRESS_CONTROL_UNIFIED*
+: This progress model indicates that the user will synchronize progressing the
+  data and control operations themselves (i.e.. this makes control interface NOT
+  thread safe). It is only valid for control progress (not data progress). If
```

Comment:
I would rephrase this to `this allows control interface to be NOT thread safe`.

Reply:
reworded